package.json

```diff
@@ -23,10 +23,11 @@
   ],
   "scripts": {
     "size": "size-limit",
-    "test": "standard && ava",
+    "test": "nyc ava tap test.js",
```

Comment:
We should still be running `standard`.

Reply:
ok i am little confused here . So far i followed https://github.com/segmentio/analytics-node/pull/148#pullrequestreview-89081587  and made it minified / simplified. So will you let me know what should i follow here? I meant should i minify it as per @vadimdemedes request OR it will be same like my previous commit?

Reply:
What I mean is we should still be running the `standard` command as part of the test.

I believe @vadimdemedes pointed to https://github.com/sindresorhus/got/blob/master/package.json#L28-L29. From there `xo` = `standard` (both are code formatting tools).

Reply:
ok got it clearly @f2prateek . I am taking care of it quickly and coming back with correct commit.